app/controllers/slugs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class SlugsController < ApplicationController
```

Comment:
Do we even need that endpoint? One benefit is that we are using Slug for Topic/Categories/Badges so consistency but on the other hand, if we just dasherize name to slug with JS - the code would be faster and simpler. What is the biggest benefit of using Slug.for?

Reply:
The main benefit is consistency between server and client, so using the same name produces the same result whether we are generating the slug on the server or client. `Slug.for` does a lot, and having to recreate this in JS will just introduce code drift. Eventually we should move those other places in the UI to do the same.

Reply:
Thank you :)